src/components/AccountPageComponents/TabContent.tsx

```diff
@@ -6,7 +6,7 @@ import BonusesTab from './BonusesTab';
 import CurrentOrdersTab from './CurrentOrdersTab';
 import OrdersHistoryTab from './OrdersHistoryTab';
 
-import { AccountTabs, tabs } from '@/constants';
+import { AccountTabs, accountTabs } from '@/constants';
```

Comment:
Нужно подправить нейминг, чем отличается AccountTabs от accountTabs не понятно. Можно, например, вот так 
```suggestion
import { AccountTabs, accountTabsData } from '@/constants';
```